launcher/LaunchController.cpp

```diff
@@ -148,8 +148,12 @@ bool LaunchController::askPlayDemo()
     return box.clickedButton() == demoButton;
 }
 
-QString LaunchController::askOfflineName(QString playerName, bool demo, bool& ok)
+QString LaunchController::askOfflineName(QString playerName, bool demo, bool* ok)
 {
+    if (ok) {
```

Comment:
Nitpick: when it's a boolean pointer it's a little confusing to coerce to a boolean 
```suggestion
    if (ok != nullptr) {
```